internal/server/v1/observations/golden/bulk_point/all_2010.json

```diff
@@ -0,0 +1,292 @@
+              "value": 37253956,
+              "facet": 1541763368
+            },
+            {
+              "date": "2010",
+              "value": 37319500,
+              "facet": 1151455814
+            },
+            {
+              "date": "2010",
+              "value": 37349363,
+              "facet": 1226172227
+            },
+            {
+              "date": "2010",
+              "value": 36637290,
+              "facet": 10983471
+            }
+          ]
+        },
+        {
+          "entity": "geoId/0649670",
+          "observations_by_facet": [
+            {
+              "date": "2010",
+              "value": 74227,
+              "facet": 2517965213
+            },
+            {
+              "date": "2010",
+              "value": 74066,
+              "facet": 1541763368
+            },
+            {
+              "date": "2010",
+              "value": 74320,
+              "facet": 1226172227
+            },
+            {
+              "date": "2010",
+              "value": 74066,
+              "facet": 2458695583
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "variable": "Count_CriminalActivities_CombinedCrime",
+      "observations_by_entity": [
+        {
+          "entity": "country/FRA"
+        },
+        {
+          "entity": "country/USA",
+          "observations_by_facet": [
+            {
+              "date": "2010",
+              "value": 10292375,
+              "facet": 2343540064
+            }
+          ]
+        },
+        {
+          "entity": "geoId/06",
+          "observations_by_facet": [
+            {
+              "date": "2010",
+              "value": 1146072,
+              "facet": 4277990144
+            }
+          ]
+        },
+        {
+          "entity": "geoId/0649670",
+          "observations_by_facet": [
+            {
+              "date": "2010",
+              "value": 1633,
+              "facet": 4277990144
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "variable": "Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity_PerCapita",
+      "observations_by_entity": [
+        {
+          "entity": "country/FRA",
+          "observations_by_facet": [
+            {
+              "date": "2010",
+              "value": 36760,
+              "facet": 2171028778
+            }
+          ]
+        },
+        {
+          "entity": "country/USA",
+          "observations_by_facet": [
+            {
+              "date": "2010",
+              "value": 48910,
+              "facet": 2171028778
+            }
+          ]
+        },
+        {
+          "entity": "geoId/06"
+        },
+        {
+          "entity": "geoId/0649670"
+        }
+      ]
+    }
+  ],
+  "facets": {
+    "10983471": {
+      "import_name": "CensusACS5YearSurvey_SubjectTables_S2601A",
+      "provenance_url": "https://data.census.gov/cedsci/table?q=S2601A\u0026tid=ACSST5Y2019.S2601A",
```

Comment:
wonder why this happens?

It looks fine here: https://datacommons.org/browser/dc/yy2t0v

Reply:
Very good find. This is only an issue in golden data due to the json marshaller. Updated the setup to not escape html chars